File: app/src/main/java/com/aslansari/hypocoin/repository/model/MarketData.java
package com.aslansari.hypocoin.repository.model;

import com.google.gson.annotations.SerializedName;

public class MarketData {

    @SerializedName("price_usd")
    public double priceUSD;
    @SerializedName("price_btc")
    public double priceBTC;
    @SerializedName("price_eth")
    public double priceETH;
}

File: app/src/main/java/com/aslansari/hypocoin/repository/model/MarketData.kt
package com.aslansari.hypocoin.repository.model

import com.google.gson.annotations.SerializedName

data class MarketData(
    @JvmField
    @SerializedName("price_usd")
    var priceUSD: Double = 0.0,

    @SerializedName("price_btc")
    var priceBTC: Double = 0.0,

    @SerializedName("price_eth")
    var priceETH: Double = 0.0,
)